js/components/skjema/periodevelger/PeriodeTom.js

```diff
@@ -63,4 +74,14 @@ PeriodeTom.propTypes = {
     onDoubleClick: PropTypes.func,
 };
 
+const PeriodeTom = (props) => {
+    return (<PeriodevelgerContext.Consumer>
+        {
+            ({ onChange }) => {
+                return <PeriodeTomComponent {...props} onChange={onChange} />;
```

Comment:
Kan du sende inn props eksplisitt her?

Reply:
Helst ikke, dette er jo egentlig bare en liten proxy for PeriodeTom som legger til onChange-funksjonen.